packages/labs/ssr-react/README.md

```diff
@@ -12,15 +12,9 @@ This package provides tools to integrate [`@lit-labs/ssr`](../ssr/README.md) wit
 
 To get React SSR to deeply render Lit components, we'll need React JSX code to call an enhanced version of `createElement()` provided by this package. The way to achieve this depends on your project configuration.
 
-### Using the Classic Runtime JSX Transform
-
-The classic JSX transform replaces JSX expressions with `React.createElement()` function calls. In the default mode, it requires that `React` is imported and available in the scope of the JSX file.
-
-This package provides a couple different ways to handle the classic runtime:
-
-#### Monkey patching `React.createElement()` (recommended)
+### Monkey patching React element creation (recommended)
 
-This package provides a module that, when imported in a server environment, has the side-effect of monkey patching `React.createElement()` to be enhanced to add the declarative shadow DOM output to registered custom elements. This can be imported at the entry point of the application before `React` is imported.
+This package provides the `@lit-labs/ssr-react/enable-lit-ssr.js` module which, when imported in a server environment, has the side-effect of monkey patching `React.createElement()` and runtime JSX functions to be enhanced to add the declarative shadow DOM output to registered custom elements. This can be imported at the entry point of the application before `React` is imported or any JSX containing Lit components is written.
```

Comment:
```suggestion
This package provides the `@lit-labs/ssr-react/enable-lit-ssr.js` module which, when imported in a server environment, has the side-effect of monkey patching `React.createElement()` and runtime JSX functions to be enhanced to add the declarative shadow DOM output to registered custom elements. This can be imported at the entry point of the application before `React` is imported or any JSX containing Lit components is evaluated.
```